src/alignment/configs.py

```diff
@@ -244,102 +245,15 @@ class SFTConfig(transformers.TrainingArguments):
     dataset_kwargs: Optional[Dict[str, Any]] = field(
         default=None, metadata={"help": "Dataset kwargs for the SFTTrainer"}
     )
-    max_seq_length: Optional[int] = field(
-        default=None,
-        metadata={"help": ("Used by TRL for reward model training, which tries to read this parameter in init.")},
-    )
-    logging_first_step: bool = field(
-        default=True,
-        metadata={"help": ("Whether to log and evaluate the first global_step or not.")},
-    )
-    optim: Optional[str] = field(default="adamw_torch")
 
 
 @dataclass
-class DPOConfig(transformers.TrainingArguments):
+class DPOConfig(trl.DPOConfig):
     """
     Arguments related to the DPO training process itself. For all parameters, see: https://huggingface.co/docs/transformers/v4.26.1/en/main_classes/trainer#transformers.TrainingArguments
     """
 
-    beta: Optional[float] = field(
-        default=0.1,
-        metadata={"help": "The beta factor in DPO loss. Higher beta means less divergence from the initial policy."},
-    )
     hub_model_revision: Optional[str] = field(
         default="main",
         metadata={"help": ("The Hub model branch to push the model to.")},
     )
-    logging_first_step: bool = field(
-        default=True,
-        metadata={"help": ("Whether to log and evaluate the first global_step or not.")},
-    )
-    max_prompt_length: Optional[int] = field(
-        default=None,
-        metadata={"help": ("For DPO, the maximum length of the prompt to use for conditioning the model.")},
-    )
-    max_length: Optional[int] = field(
-        default=None,
-        metadata={"help": ("Used by TRL for reward model training, which tries to read this parameter in init.")},
-    )
-    optim: Optional[str] = field(default="rmsprop")
-    remove_unused_columns: bool = field(default=False)
```

Comment:
Same here `remove_unused_columns` is needed since it overrides the default value in `transformers.TrainingArguments`